docs/manual/widgets.rst

```diff
@@ -578,7 +577,55 @@ When this is defined it will be used by :meth:`ListBox.__iter__` and
 
 .. currentmodule:: urwid
 
+Scrollable Widgets
+==================
+Scrollable widgets can scroll long content which normally not fit into the screen resolution.
+Scrolling is normally supported using keyboard positioning keys and mouse wheel.
+
+Comparing to the :class:`ListBox`,
+:class:`Scrollable` handles fixed and flow widgets directly instead of using list of small widgets.
+And :class:`ScrollBar` provide visual scrollbar with scrolling position information.
```

Comment:
Can ScrollBar be used with ListBox?

Reply:
Only if `ListBox` will be extended to support positioning API